src/bin/cargo-miri.rs

```diff
@@ -197,7 +227,7 @@ fn xargo() -> Command {
         // Bootstrap tells us where to find xargo
         Command::new(val)
     } else {
-        Command::new("xargo")
+        Command::new("xargo-check")
```

Comment:
It really is that easy :smile:

Reply:
Well, longer-term we'd ideally also stop creating a `Cargo.toml`. ;)